contexts/SettingsContext.tsx

```diff
@@ -1,14 +1,14 @@
 import { createContext, ReactNode, useEffect, useState } from "react";
 import Promise from "lie";
```

Comment:
Can it be deleted? I think it is imported accidentally.

Reply:
c793a979ed76a12f57c5a719f4e66d505b83c4ea